Cisco catalyst pack (#26491)
demisto/content: Packs/CiscoCatalyst/README.md

# Cisco Catalyst
This pack includes Cortex XSIAM content.

## Add timezone to the logs  

> The only supported event time is the event time with timezone.

Reviewer: ```suggestion
The only supported event time is an event time with the time zone.
```

1. Access the switch's command-line interface (CLI) using a terminal emulator or SSH.

> 2. Enter privileged EXEC mode by entering the following command and providing the enable password:

Reviewer: ```suggestion
2. Access privileged EXEC mode by entering the following command and providing the enable password:
```

```
enable
```
3. Enter global configuration mode:

```
configure terminal
```

> 4. Configure the logging timestamp and specify the desired time format with timezone:

Reviewer: ```suggestion
4. Configure the logging timestamp and specify the desired time format with the time zone:
```

```
logging timestamp datetime UTC
```
5. Exit configuration mode:
```
exit
```

> 6. to save the configuration changes run the commend:

Reviewer: ```suggestion
6. To save the configuration changes run the command:
```

```
write memory
```

> **Note** The time format will be: "May 16 2023 14:30:00 UTC"

Reviewer: ```suggestion
**Note** The time format is: "May 16 2023 14:30:00 UTC"
```
